Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:2.7
+
+ADD . /project
+WORKDIR /project
```

Comment:
Would be nicer to either use the default that the python and django onbuild images use (``/usr/src/app``) or what we use aldryn (``/app``).